paper/ms.tex

```diff
@@ -130,7 +130,7 @@
 
 \title{%
 Fast and scalable Gaussian process modeling with applications
-to astronomical time series
+to astronomical time series and power spectrum estimation
```

Comment:
I'm not sure that I want to add this to the title. I think that in most cases we don't actually _care_ about "the power spectrum". We actually care about the fundamental physical parameters of the process that generated the data. It can be useful to look at the power spectrum but that's not what we're actually after. I think that putting this in the title actually has an effect opposite the social change that I want to create :smile: